src/ProductConstructionService/ProductConstructionService.DependencyFlow/InProgressPullRequest.cs

```diff
@@ -50,4 +50,13 @@ public class InProgressPullRequest : DependencyFlowWorkItem, IPullRequest
 
     [DataMember]
     public DateTime? NextCheck { get; set; }
+
+    [DataMember]
+    public InProgressPullRequestState State { get; set; }
```

Comment:
Maybe we should rather add `Conflict` to `MergePolicyCheckResult`?

Reply:
But is it the correct place? In my opinion, if `MergePolicyCheckResult` had a state called `Conflict`, it would mean that the head of the PR branch is in conflict with the PR target branch, but that's not the case here

Reply:
Ah, right. Got it. Then keep it as is but rename the other state `Open` to `Mergeable`?

Reply:
And maybe rename the field to `MergeState`?

Reply:
Good point, done